pkg/agent/execbridge.go

```diff
@@ -34,6 +34,21 @@ func makePluginInterface(r *Runner) *execPluginInterface {
 	return &execPluginInterface{runner: r}
 }
 
+// scalingResponseType indicates type of scaling response from the scheduler plugin
+type scalingResponseType string
+
+const (
+	scalingResponseTypeDenied            = "denied"
+	scalingResponseTypeApproved          = "approved"
+	scalingResponseTypePartiallyApproved = "partiallyApproved"
+	scalingResponseTypeFailed            = "failed"
+)
+
+const (
+	upscaleDirectionLabel   = "inc"
```

Comment:
Can we somehow reuse the existing constants directionValueInc and directionValueDec?

Reply:
done